src/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImpl.Windows.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Xml
+{
+    internal partial class XmlTextReaderImpl
+    {
+        private static string ConvertAbsoluteUnixPathToAbsoluteUri(string url, XmlResolver resolver)
```

Comment:
_If_ we wanted to keep it out of the Windows build entirely, we could do it as a partial method, e.g.
```C#
partial static void ConvertAbsoluteUnixPathToAbsoluteUri(ref string url, XmlResolver resolver);
```
and then simply not provide a Windows implementation, as the url wouldn't change.